ticket-booker/src/components/Header/Header.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { HeaderContainer, TextContainer } from './Header.style';
+import { darkModeColors } from '../../constants/colors';
```

Comment:
would you consider to implement a functionality to switch between dark and light mode?

Reply:
Yes, that will be the aim for  a future sprint